### DISCUSSION TITLE
Modalita' di pagamento in documento PDF

### DISCUSSION THREAD

**Discussion by jehol75** [fattureincloud/api#102] [Q&A] [answered]
Salve,
   nella versione precedente delle API c'era il metodo di visualizzare la modalita' di pagamento nel documento PDF generato

![image](https://user-images.githubusercontent.com/2175833/193416045-0a0e24c2-fa28-47e7-80f9-cc66b04babe0.png)

adesso, nonostante venga passato la modalita' di pagamento in 2 punti (payment merhod and payment account) diversi la modalità di pagamento non appare (solo e sempre "paga con paypal")

![image](https://user-images.githubusercontent.com/2175833/193416076-013466c9-f5e4-4e06-8eed-272c2898effd.png)

Di seguito il json

{
  "data": {
    "type": "invoice",
    "entity": {
      "name": "azienda di test",
      "vat_number": "0123456789",
      "address_street": "via delle vie 12",
      "address_postal_code": "12345",
      "address_city": "BG",
      "address_province": "BG",
      "country": "Italia",
      "e_invoice": true,
      "ei_code": "AZerty2"
    },
    "number": 56,
    "numeration": "/999",
    "date": "2022-10-01",
    "currency": {
      "id": "EUR"
    },
    "language": {
      "code": "it",
      "name": "italiano"
    },
    "subject": "",
    "visible_subject": "",
    "notes": "Grazie per la preferenza accordataci.È un piacere lavorare con voi al vostro progetto.Cordiali saluti, SINFONIA cooperativa sociale",
    "payment_method": {
      "id": 1996408
    },
    "e_invoice": true,
    "ei_data": {
      "payment_method": "MP08"
    },
    "items_list": [
      {
        "code": "",
        "name": "test2",
        "category": "",
        "qty": 1.0,
        "net_price": 123.0,
        "gross_price": 150.06,
        "vat": {
          "id": 0
        },
        "discount": 0.0
      }
    ],
    "payments_list": [
      {
        "status": "not_paid",
        "due_date": "2022-10-16",
        "amount": 150.06,
        "payment_account": {
          "id": 1996408
        }
      }
    ]
  }
}

**Comment by emajo** [comment, marked as answer]
Ciao @jehol75, la differenza tra le due fatture di cui hai mandato gli screenshot sta nel campo `show_payment_method`, nel primo caso è stato settato a `true` mentre nel secondo a `false`.
